### ISSUE TITLE
More duplicate tag deployments - Project 255 & Project 153

### ISSUE THREAD

**Issue by StuMackenzie** [MotusDev/Motus-TO-DO#548]
Project 255 - GCTH with love russia

9 of 10 tags deployed on GCTH in Russia - are duplicates with Semi-sands (Project 153) deployed this spring. Obviously this is something we should have caught earlier, but didn't. I'm not sure there is anything that we can do about it since both tags have been deployed. Separating fall migration between these two species could be quite challenging in a few instances, but we'll see.

We may need to discuss ways to avoid this from happening - bigger warnings, inability to register tags at all that are already in the system, laser beams, etc.

**Comment by StuMackenzie** [comment]
The rest of the tags on this list for this project are duplicates and can be deleted.

**Comment by StuMackenzie** [comment]
The rest of the tags on this list for this project are duplicates and can be deleted.

**Comment by cjardine-bsc** [comment]
Lucas, can you send me a list of tagIDs to merge?

**Comment by leberrigan** [comment]
Sure. Just so everyone's aware, it's actually project 255 that has the GCTH!

**Comment by leberrigan** [comment]
I don't see any duplicate deployments on their project... I think we're good!
https://motus.org/data/project/tags?projectID=255

**Comment by leberrigan** [comment]
Okay this is a bit convoluted, but I think I figured it out?

Project 255: they registered 10 tags on **May 17, 2019**
Project 153: they registered 10 tags with the same parameters between **May 21, 2019** and **May 30, 2019**

[These tags show up in both project's data issues as a duplicated tag registration](https://motus.org/data/issues?id=255).  It looks like this part needs a bit of tidying. The description of this issue has no mention of the duplicated tags being registered on another project which should be highlighted. In addition, the 'view' link is broken, giving the 'No tag ID found' warning.

**Comment by StuMackenzie** [comment]
K, thanks. - I'll wait for further feedback before engaging with either project.

**Comment by cjardine-bsc** [comment]
I can tidy up the duplicate registration issue, thanks for flagging that. It actually doesn't apply to these tags though, given they are duplicate tags, not duplicate registrations. 

The error that applies in this case is the overlapping deployment error (v_tag_overlap_deploy_prop) which looks to be returning correctly. I'd say you're fine to talk with the projects and let them know that their tag detections from those tags will be ambiguous until the life span expires on one of the tags. Not much more we can do sadly, except keep working with lotek to prevent this from happening going forward.

**Comment by StuMackenzie** [comment]
Just posting this note to keep this issue alive.  This is more of an analytical task now.

Once we're relatively certain that we've received all the fall data from the continent, I'd like to take a look and try and tease these two projects apart. Most of the SESA should have migrated through before the GCTH arrived. So as rough starting point any detections after September 1 are likely GCTH, any before SESA. It's not perfect, but we can start there.

Here's hoping.

**Comment by StuMackenzie** [comment]
@leberrigan Can you look at this again and see what you think ? I'd like to try and separate the two projects as best as possible.

**Comment by StuMackenzie** [comment]
It appears that all data have been removed from the tags assigned to project 255???

**Comment by leberrigan** [comment]
To answer Stu's question above, data are currently not displayed on Motus.org because it 
includes aliased tags.

### TL;DR

**I can't conclusively identify any GCTH in this dataset.**
- Temporal-spatial distribution is inconclusive.
- Flight speed is not unusual for any true detections.
- Only tag that was not deployed in project 153 (tag 100) does not have any true detections I can pick out.
- There just isn't very much data for these tags so it's not easy to tell what's going on.
- Next step is to check all SESA deployments in project 153 from spring 2019 and see if they go to the same places as these tags.

### Background
**Project 153** _[ID: 91-99, BI: ~10.1]_ 9 SESA were trapped in Louisiana [[1](https://goo.gl/maps/UKLu7BYX9wLN3eFw8), [2](https://goo.gl/maps/FwJoH9Wzk5QS17dA9)] in **mid-May 2019** and affixed with NanoTags which had been in storage.
**Project 255** _[ID: 91-100, BI: ~10.1]_ 10 GCTH were displaced to [Russia](https://goo.gl/maps/fteYBLM7RFxj6crQ9) where they were affixed with NanoTags prior to release in **mid-June 2019**.

### Mapping Detections
Downloading data through R, we can see a broad distribution of detections, mostly between Saskatchewan and Lake Erie. 
![Proj255_map_by_month-week](https://user-images.githubusercontent.com/35931796/82901606-8fecd480-9f34-11ea-8a5c-5759c042c31c.png)
*Figure 1: Map of detections by month and week. Data only include runs with motusFilter != 1*

Early detections in June and July can be assumed to be SESA (proj 153), but detections in August are more ambiguous. 
Detections in late September are more likely to be GCTH than any other, but this still falls under the typical migratory period of SESA in that region.

![proj255_map_sept-week4](https://user-images.githubusercontent.com/35931796/82903541-7bf6a200-9f37-11ea-8cef-b0c0d47610d1.png)

![eBird_SESA-GCTH_barCharts](https://user-images.githubusercontent.com/35931796/82903410-494ca980-9f37-11ea-9dbe-5504d2cc50be.png)

If we look at the runs removed by the motusFilter we can see there all with runLen = 3 have been removed (9 runs in total). Of these detections, we know that one is certainly false (in Nova Scotia), and the others may warrant further investigation, but still fall in the migratory period overlap.
![Proj255_map_by_month-week_runLen3](https://user-images.githubusercontent.com/35931796/82903761-d09a1d00-9f37-11ea-9829-166f74ead314.png)

### Flight Speed

Examining flight speed doesn't really improve anything. We have a lot of short runs and not much of anything else. Even if we filter for runLen > 2 we don't have any suspicious flight speeds so I'm doubting there's anything in here.
![proj255_dist_by_rate](https://user-images.githubusercontent.com/35931796/82905589-6fc01400-9f3a-11ea-86e2-c321e1df6dd2.png)

### Tag 100

There is one tag that was not deployed in project 153 which has the ID 100. Unforunately, all runs associated with this tag are of length = 2 with a total of 56 runs.  These detections are clearly mostly false positives and it's really hard to tell which of those could possibly be real. I decided to look at those on Lotek receivers and they don't seem very promising. All of them have lots of SESA detections, other tags with the same burst interval present, or are in Louisiana.

### Conclusions

I don't really see a way we can reliably separate out GCTH detections from this dataset. The next step would probably be to compare all SESA detections from tags deployed at the same as those 9 from project 153 and see if they're all going to the same place.

**Comment by ajmacdonald3** [comment]
In case this helps, I made a bunch of maps a few months ago when working with SESA data including those from 153. Here's a rough map of paths of SESA detections for SESA that stopped in the Gulf of Mexico - generally birds from project 153 (2017-2019). Points represent stopovers and green is before 01 July and purple is after 01 July:
![SESA-GOM-stops-routes-map2](https://user-images.githubusercontent.com/44593593/82941476-4360c380-9f4b-11ea-9f5f-4f51700852da.png)
Here are some individual diagnostic maps I made of SESA tagged for project 153 in 2019 (none are the overlapping deployments). Red asterisks are deployment location, points are detections, dashed lines are direct flights. These are the birds I determined had true detections away from their deployment location:
![34636-map](https://user-images.githubusercontent.com/44593593/82942043-3db7ad80-9f4c-11ea-9dd1-6e34b62ae9b6.png)
![34637-map](https://user-images.githubusercontent.com/44593593/82942047-3ee8da80-9f4c-11ea-9c67-c4b010a368b6.png)
![34640-map](https://user-images.githubusercontent.com/44593593/82942052-3f817100-9f4c-11ea-9614-d50ab6e5ee4d.png)
![34977-map](https://user-images.githubusercontent.com/44593593/82942057-401a0780-9f4c-11ea-93d7-556229ac18ae.png)
![34985-map](https://user-images.githubusercontent.com/44593593/82942061-40b29e00-9f4c-11ea-8711-ec7bbdf65059.png)
![34988-map](https://user-images.githubusercontent.com/44593593/82942062-40b29e00-9f4c-11ea-82b2-47ad4b3623ef.png)
![34992-map](https://user-images.githubusercontent.com/44593593/82942064-414b3480-9f4c-11ea-866e-04fd5534f9de.png)
![35000-map](https://user-images.githubusercontent.com/44593593/82942067-41e3cb00-9f4c-11ea-9d77-635dfc99605e.png)
![35007-map](https://user-images.githubusercontent.com/44593593/82942069-41e3cb00-9f4c-11ea-9e2b-cc2a6abb16c4.png)
![35008-map](https://user-images.githubusercontent.com/44593593/82942071-427c6180-9f4c-11ea-998f-3b542289a9da.png)
![35009-map](https://user-images.githubusercontent.com/44593593/82942073-4314f800-9f4c-11ea-8734-85b9370d4d27.png)
![35011-map](https://user-images.githubusercontent.com/44593593/82942075-4314f800-9f4c-11ea-85d6-ac4cdf7793a4.png)
![35013-map](https://user-images.githubusercontent.com/44593593/82942077-43ad8e80-9f4c-11ea-9890-4adfc9708a03.png)
![35014-map](https://user-images.githubusercontent.com/44593593/82942079-44462500-9f4c-11ea-84bd-45557b651491.png)
![35015-map](https://user-images.githubusercontent.com/44593593/82942081-44462500-9f4c-11ea-991e-f6557b5811ba.png)
![35016-map](https://user-images.githubusercontent.com/44593593/82942083-44debb80-9f4c-11ea-836d-699398918c9d.png)
